packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -288,19 +295,45 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
     );
   }
 
+  // Adjusts the margin percentage, based on the width of the svg (inversly propotional)
+  private _adjustBarSpacing(): void {
+    const svgWidth = this.barChartSvgRef.current?.getBoundingClientRect().width;
+    if (svgWidth) {
+      const BAR_SPACING_FACTOR = 200;
+      const currentBarSpacing = clamp(BAR_SPACING_FACTOR / svgWidth, 1, 0.2);
```

Comment:
Clamp documentation says 2nd argument is min and 3rd is max.
clamp = (value: number, min: number, max: number)

You have used it other way. Is it intentional.

Reply:
![image](https://github.com/microsoft/fluentui/assets/70366079/7810eacd-efa6-4e38-891f-b6cfd77db5f6)

The clamp function I've used is coming from fluentui/react